### PR TITLE
Add docker compose and other configurations.

### PR DESCRIPTION
**Pull request by amalpaul54111**

### CODE REVIEW

**Comment by amalpaul54111** [comment]
@jeffw16 `./_sources/configs/default.vcl` is used by the varnish container in the `docker-compose.yml`. Or should we just go with the existing location of that file? I would like to hear your opinion on that. 

Should we also add the existing `readme.md` from the **Canasta** repo? 
And the backup scripts would be important too right?
Are there anymore files we could add?

**Comment by jeffw16** [comment]
@amalpaul54111 Good catch. Let's copy the `default.vcl` file to here and change that to point to `./configs/default.vcl`.

No need to add the existing README.

Let's not worry about the backups for now. We're going to try to use Restic soon to generate some actual backups.

Let's go ahead and just merge these - you got basically everything. If we need more, I can add them.

Thank you very much Amal, it's really very much appreciated. :)

File: default.vcl [hunk 1]
```diff
@@ -0,0 +1,149 @@
+vcl 4.0;
```
Comment (jeffw16) [inline default.vcl:1]: Let's move it to under `config/`

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      - "${HTTPS_PORT:-443}:443"
+    volumes:
+      - caddy-data:/data
+
+  varnish:
+    image: varnish:7.0.2-alpine
+    restart: unless-stopped
+    volumes:
+      # - ./_sources/configs/default.vcl:/etc/varnish/default.vcl:ro
+      - ./default.vcl:/etc/varnish/default.vcl:ro
```
Comment (jeffw16) [inline docker-compose.yml:69]: `./config/default.vcl:/etc/varnish/default.vcl:ro`

**Review by jeffw16 — CHANGES_REQUESTED** [review]

File: default.vcl (hunk 1 above)
Comment (amalpaul54111) [reply default.vcl]: Ok that's good, Done!

File: docker-compose.yml (hunk 2 above)
Comment (amalpaul54111) [reply docker-compose.yml]: Okay. Done!

**Review by jeffw16 — APPROVED** [review]
LGTM

**Pull request merged**